src/scripts/main.js

```diff
@@ -1,3 +1,59 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('.field');
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+
+const maxCount = 10;
+const minCount = 2;
+const rows = table.rows;
+
+function updateButtonStatus() {
```

Comment:
I would personally rather than calling this everywhere, just run this function on clicking on the document body
```js
document.body.addEventListener("click", updateButtonStatus);
```

Reply:
ok thank you. It's a really good advice! I've already changed it.